packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+| userId      | `id` of a User                                               | URL          | String     | TRUE     |
+
+###### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/catalog/apps/amazon_aws"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+##### Add an App target to an App administrator role given to a Group
+
+<ApiOperation method="put" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/catalog/apps/${appName}" />
+
+Adds an App target for an `APP_ADMIN` Role assigned to a Group
+
+When you add the first App target, you reduce the scope of the role assignment. The Role no longer applies to all App targets, but applies only to the specified target.
+
+Adding an App target overrides any existing instance targets of the app. For example, if someone was assigned to administer a specific Facebook instance, calling this endpoint with `facebook` for `appName`, would make them administrator for all Facebook instances.
+
+###### Request parameters
+
+
+| Parameter   | Description                                                  | Param Type   | DataType   | Required |
+| :---------- | :----------------------------------------------------------- | :----------- | :--------- | :------- |
+| groupId     | `id` of a Group                                              | URL          | String     | TRUE     |
+| roleId      | `id` of a Role                                               | URL          | String     | TRUE     |
+| appName     | `name` of App target from catalog to scope role assignment   | URL          | String     | TRUE     |
+
+###### Response parameters
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+###### Request example
+
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/IFIFAX2BIRGUSTQ/targets/catalog/apps/amazon_aws"
+```
+
+###### Response example
+
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Add an App Instance target to an App administrator role
+
+Assign an admin role to a specific App Instance to grant the admin permission to manage an instance of one App or instances of multiple Apps. App Instances are specific Apps that admins have created in their org. For example, there may be a Salesforce App configured differently for each sales region of a company. When you create an App Instance target, an admin may be assigned to manage only two instances of the configured Salesforce Apps and perhaps assigned to manage an instance of another configured App, such as Workday.
```

Comment:
and perhaps assigned > and perhaps are assigned
remove comma before "such as Workday"